DataFormats/HcalDetId/interface/HcalZDCDetId.h

```diff
@@ -47,20 +52,22 @@ class HcalZDCDetId : public DetId {
 
   static bool validDetId( Section se, int dp ) ;
 
-   private:
+private:
 
-      enum { kDepEM  = 5,
-	     kDepHAD = 4,
-	     kDepLUM = 2,
-	     kDepTot = kDepEM + kDepHAD + kDepLUM };
+  enum { kDepEM  = 5,
+	 kDepHAD = 4,
+	 kDepLUM = 2,
+	 kDepRPD = 16,
+	 kDepRun1= kDepEM + kDepHAD + kDepLUM,
+	 kDepTot = kDepRun1 + kDepRPD};
 
-   public:
+public:
 
-      enum { kSizeForDenseIndexing = 2*kDepTot } ;
+  //  enum { kSizeForDenseIndexing = 2*kDepTot } ;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant